docs/developer/export-products.mdx

```diff
@@ -0,0 +1,353 @@
+---
+title: Exporting Products Information
+sidebar_label: Exporting products
+---
+
+import Chart from "@site/components/Chart";
+
+## Introduction
+This guide describes how to export products from the Saleor GraphQL API.
+Exporting products can be useful for creating backups of your data or for easy
+and quick bulk editing.
+
+You can export all products, filtered products, or products with specific IDs. 
+Products data can be exported to CSV or XLSX file, but CSV file is recommended because
+it is much less time-consuming to generate. You can also define which fields will be exported.
+If any product variants fields are specified then products and variants data are exported.
+Link to download file is sent by email to the requestor. If any error occurs then
+email with information about problems is sent.
+
+:::note
+When the export is made by the `App` email is not sent. You can get the file by query the
+`ExportFile` with ID returned from `exportProducts` mutation.
+:::
+
+### Export file structure
+Each row in the exported file represents a single product variant, but it also contains
+general product fields.
+For example, if a product has three variants, there will be three lines in total.
+Each line will contain common product fields such as name or description,
+and fields specific to given variant:
+
+| id | name         | description                    | variant sku | variant weight |
+|----|--------------|--------------------------------|-------------|----------------|
+| 61 | Orange Juice | No additives, no preservatives | 12345       | 500 g          |
+| 61 | Orange Juice | No additives, no preservatives | 32145       | 1000 g         |
+| 61 | Orange Juice | No additives, no preservatives | 45123       | 1500 g         |
+
+The exact shape of the file depends on the fields selected for export
+(see the `exportInfo` input fields). If no variant fields are provided,
+each row will contain only product fields:
+
+| id  | name         | description                    | category    | product weight |
+|-----|--------------|--------------------------------|-------------|----------------|
+| 61  | Orange Juice | No additives, no preservatives | juices      | 500 g          |
+| 65  | Apple Juice  | No additives, great taste      | juices      | 1000 g         |
+| 101 | Polo Shirt   | Nice and comfortable           | polo-shirts | 200 g          |
+
+### The charts below explains workflow of the exporting products:
```

Comment:
One last suggestion - titles should be concise and rather short, so I would move this sentence to a paragraph and change the title.

```suggestion
### Workflow

The charts below explain the internal workflow of exporting products:
```